inst/cpp/sim.cpp

```diff
@@ -71,7 +83,6 @@ List sim_wrapper_cpp (NumericVector A, List design, List par, NumericVector iov_
   int start;
   memset(rate, 0, sizeof(rate));
   // insert observation compartment
-  // insert bioavailability definition
```

Comment:
Unrelated. This line was actually not used, the grep function was looking for `// insert bioav definition`, so never picked up on this one. A little down in the code it was written correctly. So it seems this line was not functional, therefore can be removed.